toolchain/lowering/lowering_context.cpp

```diff
@@ -124,17 +140,27 @@ auto LoweringContext::BuildFunctionDefinition(SemIR::FunctionId function_id)
   LoweringFunctionContext function_lowering(*this, llvm_function);
 
   const bool has_return_slot = function.return_slot_id.is_valid();
-  const int first_param = has_return_slot ? 1 : 0;
 
   // Add parameters to locals.
+  // TODO: This duplicates the mapping between semantics nodes and LLVM
+  // function parameters that was already computed in BuildFunctionDeclaration.
+  // We should only do that once.
   auto param_refs = semantics_ir().GetNodeBlock(function.param_refs_id);
+  int param_index = 0;
```

Comment:
The purpose of the enumerate on 154 was presumably to handle this, is the enumerate still needed under this approach?